src/app/order/OrderStatus.tsx

```diff
@@ -26,13 +28,26 @@ const OrderStatus: FunctionComponent<OrderStatusProps> = ({
             { method: 'Stripe (SEPA)', value: 'sepa_link_text' },
             { method: 'OXXO (via Checkout.com)', value: 'oxxo_link_text' },
             { method: 'Boleto Bancário (via Checkout.com)', value: 'boleto_link_text' },
+            { method: checkoutcomSEPAMethod, value: 'mandate_text_only' },
         ];
 
         const mandateText = Mandates.find(pair => pair.method === order?.payments?.[0].description);
 
         return mandateText ? mandateText.value : 'mandate_link_text';
     }, [order]);
 
+    const showMandateAsTextOnly = useCallback(() => {
```

Comment:
What's the benefit of deferring the execution of this function?

We only seem to use it once and within the same scope as its creation.

`textOnlyMethods` seems to be constant, so we can move it out of this FC as you've done with `checkoutcomSEPAMethod`.

`order?.payments?.[0].description` seems to be used various other places within this component, so we could build a shared reference to it:

```
const mandateProvider = `${order?.payments?.[0].description}`
```

which all should let us simplify this function to just:

```
const showMandateAsTextOnly = TEXT_ONLY_METHODS.includes(mandateProvider);
```